validator/sawtooth_validator/networking/handlers.py

```diff
@@ -252,6 +252,14 @@ def handle(self, connection_id, message_content):
             message_type=validator_pb2.Message.PING_RESPONSE)
 
 
+class PingResponseHandler(Handler):
```

Comment:
Could you add a doc string (or in the commit message) that gives some reasoning for this handler?

Reply:
added it under hander to match the other handler doc string and updated the commit message